lib/index.js

```diff
@@ -166,4 +178,31 @@
     return utils.applyTransform(buffer, utils.messageTransform(transform));
   };
 
+
+//~api~
+//----
+//### .timetagToDate(ntpTimeTag)
+// Convert a timetag array to a JavaScript Date object in your local timezone.
+//
+// Received OSC bundles converted with `fromBuffer` will have a timetag array:
+// [secondsSince1970, fractionalSeconds]
+// This utility is useful for logging. Accuracy is reduced to milliseconds,
+// but the returned Date object also has `fractionalSecondsInt` and `fractionalSecondsFloat` set if you need full accuracy (0.00000000023283 second, or 2^32 per second)
```

Comment:
There are serious performance implications to glomming extra fields onto existing structures in at least V8: (http://ariya.ofilabs.com/2012/02/javascript-object-structure-speed-matters.html).  I wouldn't add these fields, as they are already available in the timetag.